examples/package.json

```diff
@@ -15,6 +15,8 @@
     "@heroicons/react": "^2.0.18",
     "@react-three/drei": "^9.80.2",
     "@react-three/fiber": "^8.13.6",
+    "@tldraw/editor": "=2.0.0-canary.c7ae756c0b81",
```

Comment:
these need to be pinned, otherwise there are interface changes and I don't know enough tldraw to debug them.